src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -3994,12 +3994,25 @@ CorInfoIntrinsics ZapInfo::getIntrinsicID(CORINFO_METHOD_HANDLE method,
     CorInfoIntrinsics intrinsicID = m_pEEJitInfo->getIntrinsicID(method, pMustExpand);
 
 #if defined(TARGET_X86) || defined(TARGET_AMD64)
```

Comment:
`m_pEEJitInfo->getIntrinsicID` will return `CORINFO_INTRINSIC_Illegal` for Ceiling and Floor already given the other changes. This change will have no affect. It may be best to move this logic into the JIT.